js/index.js

```diff
@@ -1,7 +1,57 @@
 // Iteration 1: Names and Input
-// 
+
+let hacker1 = "Juliana"; 
+console.log (`"The driver's name is ${hacker1}".`);
+
+let hacker2 = "Larissa"; 
+console.log(`"The navigator's name is ${hacker2}".`);
+
 
 // Iteration 2: Conditionals
 
+if (hacker1.length > hacker2.length){
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters.`)
+} else if (hacker1.length < hacker2.length){
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`)
+} else if (hacker1.length = hacker2.length ){
+    console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`)
+}
+
+
+// Iteration 3: Loops
+
+//3.1
+
+let newNameSpace = " ";
+
+for (let i = 0; i < hacker1.length; i++){
+    newNameSpace = newNameSpace + hacker1[i] + " "
+    
+} 
+
+console.log(newNameSpace.toUpperCase() + "")
+
+
+
+//3.2
+for (let j = hacker2.length;  j >= 0 ; j--){
+    console.log(hacker2[j])
```

Comment:
cuidado, o índice sempre termina um a menos que o comprimento que do que vc quer
ou seja: se hacker2 tiver 7 letras, a ultima letra tem o índice 6 (pois começamos pelo índice 0!)
então se pegarmos `hacker2[j]` quando `j === 7`, issonão retornará a letra que queremos e sim `undefined`!

outra coisa, aqui vc vai imprimir o nome da seguinte forma:
```javascript
a
s
s
i
r
a
L
```
quando na verdade, deveria ser:
```javascript
assiraL
```

uma sugestão q dou é fazer parecido com a iteração anterior, e "armazenar" as letras numa variável nova e só depois de completar o ciclo do for, dar o `console.log()` dessa outra variável

Reply:
Dani, eu tava vendo aqui, esses métodos que acabei fazendo agora e apesar de dizer que não é o mais performático, achei interessante aprendê-los. Mas vou fazer também novamente com o loop for. 

let newNameOrder = hacker2.split("").reverse().join("");
console.log(newNameOrder); 

Um método então quebra, o outro reverte (ao contrário) e o último junta tudo novamente, certo? Funcionou.

Reply:
let newNameOrder = "";

for (let j = hacker2.length - 1; j >= 0; j--){  
    newNameOrder = newNameOrder + hacker2[j];
}

console.log(newNameOrder);